packages/storage-plugin/src/storage.plugin.ts

```diff
@@ -30,7 +37,18 @@ export class NgxsStoragePlugin implements NgxsPlugin {
             val = {};
           }
 
-          if (key !== '@@STATE') {
+          if (options.migrations) {
+            options.migrations.forEach((strategy: MigrationStrategy) => {
+              const versionMatch = strategy.version === getValue(val, strategy.versionKey || 'version');
+              const keyMatch = (!strategy.key && isMaster) || strategy.key === key;
+              if (versionMatch && keyMatch) {
+                val = strategy.migrate(val);
+                hasMigration = true;
```

Comment:
can't we just assume that hasMigration is true if there are migrations in the list?

Reply:
What if no migrations matched? No reason to set it then.